scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -439,7 +439,6 @@ object Defn {
 
 @ast class Pkg(ref: Term.Ref, stats: List[Stat]) extends Member.Term with Stat {
   checkFields(ref.isQualId)
-  checkFields(stats.forall(_.isTopLevelStat))
```

Comment:
This is no longer true for packages in Scala 3 and we have no way of checking that here. However this is still checked while parsing.